source/config.civet

```diff
@@ -9,7 +9,7 @@ configFileNames := new Set [
   "civetconfig.civet"
 ]
 
-function findInDir(dirPath: string): Promise<string | undefined>
+function findInDir(dirPath: string): Promise<string | ???>
```

Comment:
`???` means `unknown`, this isn't what findInDir is returning. The `undefined` return value indicates not having found a config file.